src/operators/frei0r_op.ml

```diff
@@ -134,9 +134,10 @@ class frei0r_mixer ~kind ~name bgra instance params (source : source) source2 =
             let rgb' = Content.Video.get_data rgb' in
             for i = offset to offset + length - 1 do
               (* TODO: we could try to be more efficient than converting to/from RGBA32 and swap colors... *)
-              let img = Video.get rgb i in
+              let img = Video.Canvas.render rgb i in
               let img = Image.YUV420.to_RGBA32 img in
-              let img' = Video.get rgb' i in
+              let img' = Video.Canvas.get rgb' i in
```

Comment:
What's the difference between `render` and `get`?

Reply:
`get` gives you a canvas (= a formal sum of images) whereas `render` computes an image from the canvas (it computes the result of the formal sum of images).